src/components/FeatureTrainingModal.tsx

```diff
@@ -93,6 +93,9 @@ type BaseFeatureTrainingModalProps = {
 
     /** Modal width */
     width?: number;
+
+    /** Whether to disable the modal */
```

Comment:
I find this comment a bit confusing. Why is a variable called `isVisible` whether to disable the modal?

Reply:
Yes, that makes more sense to me. Thanks.